config/helm/chart/default/tests/Common/csi/clusterrole-csi_test.yaml

```diff
@@ -24,7 +27,8 @@ tests:
 
   - it: should have the correct rules
     set:
-      image: image-name
+      platform: kubernetes
+      image: image-nam
```

Comment:
does not matter for the test, and because it's a cherry pick you don't have to change it

But I guess it should be `image-name`